.github/workflows/prod-efax-service-build-deploy.yaml

```diff
@@ -3,13 +3,19 @@ name: PROD Deploy of Efax Webservice Image
 on:
   workflow_dispatch:
     branches: [main]
```

Comment:
Now that there is a tag input, is this line still needed?

Reply:
We do not, and I removed it.